src/applications/static-pages/sass/static-pages.scss

```diff
@@ -77,3 +77,10 @@
     display: none;
   }
 }
+
+
+// Makes anything look like an h3
+.h3 {
```

Comment:
i wasn't _exactly_ sure how to approach this - will there be more that need this style? do i need to add the font-family? it works for this immediate need

Reply:
You might see if the `heading-level-3` class works for this.